tefla/core/eval_metrices.py

```diff
@@ -0,0 +1,448 @@
+import numpy as np
```

Comment:
@mkulariya1 format this code using `yapf` and also check if there are any `pep8` and `pylint` errors.  We use two space as indent width.

Reply:
done